util/dvsim/Modes.py

```diff
@@ -422,7 +422,7 @@ def __init__(self, regdict):
         self.type = ""
         if not hasattr(self, "mname"):
             self.mname = "regression"
-        self.tests = []
+        self.tests = None
```

Comment:
Could this gain a comment explaining about the 3 stages ("None, empty list, some tests")?